frontend_v2/src/styles/base.scss

```diff
@@ -1011,165 +1011,7 @@ ul {
 .container {
   position: relative;
   z-index: 1;
-  padding: 15px 40px;
```

Comment:
padding-top has been changed to 64 px on all devices because the nav bar is 64 px on all devices.